src/backend/src/prisma/seed-data/teams.seed.ts

```diff
@@ -90,4 +93,15 @@ const financeTeam = (headId: number): Prisma.TeamCreateArgs => {
   };
 };
 
-export const dbSeedAllTeams = { ravens, orioles, justiceLeague, huskies, avatarBenders, plLegends, financeTeam };
+const meanGirls = (headId: number): Prisma.TeamCreateArgs => {
+  return {
+    data: {
+      teamName: 'Mean Girls',
```

Comment:
Ok damn, I love the Mean Girls, could you rename the team name to Slack Testing or smthn, and then we can keep all the mean girls stuff in the desc/ members